markersize validator extended

ugorsahin opened the pull request:
The current markersize argument in the plot function only allows the markersize to be an instance of float or integer. Thus, it restricts the usage of size in scatter plots. Extending the validator with pd.Series and np.ndarray help us to utilize the underlying feature. Here is a result using pd.Series for markersize

![stock_price](https://user-images.githubusercontent.com/37257227/210429743-1c6afcc8-9c05-49fc-b1ed-0a49cd4ac3a7.png)

DanielGoldfarb commented:
@ugorsahin 
Uğur,  Thank you very much for contributing to mplfinance.   Can you please post the code that you used to make [the example plot above](https://user-images.githubusercontent.com/37257227/210429743-1c6afcc8-9c05-49fc-b1ed-0a49cd4ac3a7.png) or, if you have time, add the example to the [addplot tutorial](https://github.com/matplotlib/mplfinance/blob/master/examples/addplot.ipynb).  Thank you.

ugorsahin commented:
@DanielGoldfarb 

Hello Daniel. I added an example to the [addplot tutorial](https://github.com/ugorsahin/mplfinance/blob/markersize_validator_fix/examples/addplot.ipynb). The cell number is 19. I scaled the values to better show variable marker sizes and added an explanation

![variable_scatter](https://user-images.githubusercontent.com/37257227/210848726-7f939685-9a4d-453e-8476-93f8f5c97040.png)
